homeassistant/components/sensor/dominionenergy.py

```diff
@@ -0,0 +1,89 @@
+"""
+Support for Washington Dominion Energy.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.dominion_energy/
+"""
+
+import logging
+from datetime import timedelta
+import voluptuous as vol
```

Comment:
Add a blank line above this line line. Python internal libraries, 3rd party global libraries (like voluptuous) and Home Assistant internal imports should each be in a code block separated by a blank line.

Reply:
Thank you ! i did change that